charts/varnish/templates/configmap-nginx.yaml

```diff
@@ -4,5 +4,38 @@ metadata:
   name: {{ include "varnish.fullname" . }}-nginx
   labels:
     {{- include "varnish.labels" . | nindent 4 }}
+{{- if .Values.nginx.directorCustomConfig -}}
+data:
+  nginx.conf: |-
+    events {}
+    http {
+      server { # simple reverse-proxy
+        listen       81;
+        location /health {
+          return 200 'status: ok';
+          access_log off;
+        }
+        # pass requests for dynamic content to rails/turbogears/zope, et al
+        location / {
+          rewrite /next-gen/(.*) /$1  break;
+          resolver kube-dns.kube-system.svc.cluster.local ipv6=off;
+          proxy_pass https://$http_x_goto;
+          proxy_set_header Host $http_x_goto;
+        }
+        location /director/ {
+          proxy_pass http://cdn-{{ .Values.nginx.cdnType }}.cresta.com/director-{{ .Values.nginx.directorVersion }}/;
```

Comment:
Not sure if this will work because the template language is being used to modify a [multiline yaml](https://yaml-multiline.info/).  I couldn't find explicit note in the [helm chart template guide doc](https://helm.sh/docs/chart_template_guide/function_list/) that says for sure this will work or not.. More testing needed.

Reply:
I hope it would work :D But the path is not correct - we are publishing to `/director/<version>` (https://github.com/cresta/director/blob/main/.github/workflows/release-frontend.yml#L33)